### DISCUSSION TITLE
Linear Temporal Logic over finite traces

### DISCUSSION THREAD

**Discussion by johnryan465** [outlines-dev/outlines#232] [Feature requests]
For some circumstances LTLf would be more convient to specify than regex. LTLf can be converted to a DFA with a library such as https://github.com/whitemech/LTLf2DFA .

**Comment by rlouf** [comment]
Thanks for the suggestion! That would be a substantial addition (half of `parsing.py` file) to the library. Would you be willing to work on this?

**Reply by johnryan465** [reply]
Would be wiling to work on it!

**Reply by rlouf** [reply]
By all means!

**Reply by brandonwillard** [reply]
Yeah, that we be really great!
